ios-target/Model/MatchListItem.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MatchConversation.swift
+//  ios-target
+//
+//  Created by Sol Manini on 7/5/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+
+struct MatchListItem: Codable {
+  let matchId: Int?
+  let user: MatchedUser?
+  let lastMessage: String?
+  let topicIcon: URL?
+  let unreadMessages: Int?
+  
+  private enum CodingKeys: String, CodingKey {
```

Comment:
forgot to comment this on the previous PRs, but if you are just using `CodingKeys` to convert from snake to camel you can use this at decoder level: https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase
So you end up with something like:
```swift
extension JsonDecoder {
  static func camelCaseDecoder() -> JsonDecoder {
    let decoder = JsonDecoder()
    decoder.keyDecodingStrategy = .convertFromSnakeCase
  }
}

let decoder = JsonDecoder.camelCaseDecoder()
if let parsedMatch = try? .decode(MatchListItem.self, from: matchItem) {
    parsedMatchListItems.append(parsedMatch)
}
```

Reply:
great!! will try this, thanks!